dataactcore/migrations/versions/ee70a910db2e_adding_primaryplaceofperformancescope_.py

```diff
@@ -0,0 +1,41 @@
+""" Adding PrimaryPlaceOfPerformanceScope to FABS
+
+Revision ID: ee70a910db2e
+Revises: 505f4b28d33d
+Create Date: 2019-12-06 17:51:18.995586
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'ee70a910db2e'
+down_revision = '505f4b28d33d'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+
+
+def upgrade_data_broker():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('published_award_financial_assistance', sa.Column('place_of_performance_scope', sa.Text(),
```

Comment:
While we're already editing, just a general thing, we try to keep most of these on the same line (linting need not apply to migrations)

Reply:
TIL linting doesnt apply to migrations 👍